Makefile

```
@@ -147,10 +147,17 @@ help:
 	fi
 	@echo
 
+### check-rust : check if Rust is installed in the environment
+.PHONY: check-rust
+check-rust:
+	@if ! which rustc >/dev/null; then \
+		echo "ERROR: Rust is not installed. Please install Rust before continuing." >&2; \
+		exit 1; \
+	fi;
```

```suggestion
	fi;

```
Need two blank line between targets. :)